edgeimpulse-ce-manifest-fv2.xml

```diff
@@ -1,14 +1,14 @@
 <apps version="2.0">
-  <app keywords="psoc6,edgeimpulse,edgeml,ml,ai,motion,gesture" req_capabilities_v2="[cy8ckit_062s2_43012]">
+  <app keywords="psoc6,edgeimpulse,edgeml,ml,ai,motion,gesture" req_capabilities_v2="[cy8ckit_062s2_43012, cy8ckit_062_ble]">
     <name>EdgeImpulse Continuous Motion recognition</name>
     <category>Machine Learning</category>
     <id>edgeimpulse-example-continuous-motion-recognition</id>
     <uri>https://github.com/edgeimpulse/mtb-example-edgeimpulse-continuous-motion</uri>
     <description><![CDATA[This code example demonstrates machine learning application using EdgeImpulse and the PSoC™ 6 MCU with AIROC™ CYW43xxx Wi-Fi & Bluetooth® combo chips.
-    <br><br>For more details, see the <a href="https://github.com/edgeimpulse/mtb-example-edgeimpulse-continuous-motion/blob/main/README.md">README on GitHub</a>.<br><br><a href="https://edgeimpulse.com/">Edge Impulse</a> is the leading development platform for machine learning on edge devices, free for developers and trusted by enterprises..<br> > <a href="https://docs.edgeimpulse.com/docs/~/revisions/9dKGATK7EnFnYNRMlqcb/development-platforms/officially-supported-mcu-targets/infineon-cy8ckit-062s2">Get Started</a><br> > <a href="https://edgeimpulse.com/contact">Talk to Support</a>]]></description>
+    <br><br>For more details, see the <a href="https://github.com/edgeimpulse/mtb-example-edgeimpulse-continuous-motion/blob/main/README.md">README on GitHub</a>.<br><br><a href="https://edgeimpulse.com/">Edge Impulse</a> is the leading development platform for machine learning on edge devices, free for developers and trusted by enterprises..<br> > <a href="https://docs.edgeimpulse.com/docs/development-platforms/officially-supported-mcu-targets/infineon-cy8ckit-062s2">Get Started</a><br> > <a href="https://edgeimpulse.com/contact">Talk to Support</a>]]></description>
```

Comment:
since we have two kits lets point to the top level device documentation page (official support devices), it will save us from having to update links in the future or adding too many than what would look good inside mtb